Default.sublime-keymap

```diff
@@ -50,23 +50,46 @@
     },
     {
         "keys": ["."],
-        "command": "gs_inline_diff_goto_next_hunk",
+        "command": "gs_inline_diff_navigate_hunk",
+        "args": { "forward": true },
         "context": [
             { "key": "setting.command_mode", "operator": "equal", "operand": false },
             { "key": "setting.git_savvy.inline_diff_view", "operator": "equal", "operand": true }
         ]
     },
     {
         "keys": [","],
-        "command": "gs_inline_diff_goto_previous_hunk",
+        "command": "gs_inline_diff_navigate_hunk",
+        "args": { "forward": false },
+        "context": [
+            { "key": "setting.command_mode", "operator": "equal", "operand": false },
+            { "key": "setting.git_savvy.inline_diff_view", "operator": "equal", "operand": true }
+        ]
+    },
+    {
+        "keys": ["down"],
+        "command": "gs_inline_diff_navigate_hunk",
+        "args": { "forward": true },
+        "context": [
+            { "key": "setting.command_mode", "operator": "equal", "operand": false },
+            { "key": "setting.git_savvy.arrow_keys_navigation", "operator": "equal", "operand": true },
+            { "key": "setting.git_savvy.inline_diff_view", "operator": "equal", "operand": true }
+        ]
+    },
+    {
+        "keys": ["up"],
```

Comment:
I think we should not activate `gs_inline_diff_navigate_hunk` for `up` and `down` keys in inline diff view. The original up and down keys can be useful in some cases in inline diff view.

Nevertheless, if we want to activate them, we need to execute
```
self.view.run_command("gs_handle_arrow_keys")
```